### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by jekuaitk**
https://jira.itkdev.dk/browse/MUHU-190

Release 2.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+## Released
```
Comment (rimi-itk) [inline CHANGELOG.md]: Is this part of “Keep a Changelog”?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jekuaitk) [reply CHANGELOG.md]: According to https://keepachangelog.com/en/1.0.0/ it is.

Comment (rimi-itk) [reply CHANGELOG.md]: Is `## Released` a part of “Keep a Changelog”?

Comment (jekuaitk) [reply CHANGELOG.md]: Fixed.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**